calico/felix/config.py

```diff
@@ -163,6 +166,14 @@ def validate_cfg(self):
         self.validate_addr("PluginAddress", self.PLUGIN_ADDR)
         self.validate_addr("ACLAddress", self.ACL_ADDR)
 
+        #*********************************************************************#
+        #* Bind address must be * or an IP address.                          *#
+        #*********************************************************************#
+        if (self.BIND_ADDR != "*" and
+                not common.validate_ipv4_addr(self.BIND_ADDR)):
```

Comment:
We do not support IPV6 for `0MQ`. If we were to do so, then this would have to change (the point is that `0MQ` is IPv4 only unless you explicitly enable IPv6 by disabling the `ZMQ_IPV4ONLY` option (which none of our code does). That means that an IPv6 address wouldn't work, so we should not permit it.